css/style.css

```diff
@@ -0,0 +1,115 @@
+/*style.css*/
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+.clearfix:after {
+     content: ".";
+     clear: both;
+}
+
+.clear {
+  clear: both;
+}
+
+.alignright {
```

Comment:
Just for clarity's sake, when two words are run together they're either camel-cased `alignRight` or separated with a character, e.g. `align-right`